### DISCUSSION TITLE
Fetch all throwing a cygwin warning "Couldn't compute FAST_CWD pointer"

### DISCUSSION THREAD

**Discussion by rrodriguez1975** [gitextensions/gitextensions#9883] [Q&A]
### Environment

- GitExtensions version: 3.5.4.12724
- GIT version: 2.33.0.windows.2
- OS version: Windows 11 Professional x64 (Microsoft Windows NT 10.0.22000.0)
- .NET version: NET Framework 4.8.4470.0

### Issue description

I am using the standalone version of Git Extensions (not that integrated as an extension within Visual Studio).

When I fetch all, it is done successfully but a warning message appears regarding to cygwin which is the following:

Cygwin WARNING: Couldn't compute FAST_CWD pointer. This typically occurs if you're using an older Cygwin version on a newer Windows. Please update to the latest available Cygwin version from https://cygwin.com/. If the problem persists, please see https://cygwin.com/problems.html

I am using a Windows 11 virtual machine x64 and I have not installed cygwin in my system. I do not understand why this warning message appears constantly.

Anyone could explain me the cause?

### Steps to reproduce

Perform either Fetch or Fetch All

### Did this work in previous version of GitExtensions?

I haven't checked in older versions

**Comment by pmiossec** [comment]
I don't have real answer to your problem but looking at the error message (and because you use Win11) , I would try to install the last version of Git for Windows (that ship and use a version of cygwin):

https://github.com/git-for-windows/git/releases

This version seems to fix your issue: https://github.com/git-for-windows/git/releases/tag/v2.33.1.windows.1

**Reply by devlie** [reply]
For what it's worth, I'm seeing the same error message in git client console (version 2.40.1).

**Comment by aldwgiovanni** [comment]
maybe it could help or not but im getting the same issue whatever eand wherever i tried to fetch, when opening Github desktop, it ask me to push my branch, but when on Visual Studio it show the number of unpushed commit. But, apart local commits, both of them refuse to fetch.
